README.md

```diff
@@ -177,37 +180,36 @@ Steppers can exert damaging forces, even if they are moving slow. If in doubt, s
   - Often, the header file `template_I2C.h`  will very closely resemble the header file of the library that you are addressing on the target device's side.
   - The implementation  `template_I2C.cpp`, however, looks quite different: It will do nothing else but "wrap" each function's arguments into a command, transmit it to the target, and, optionally, receive the target device's reply.
-- `template_I2C.h`  and `template_I2C.cpp` - controller library templates. Its main function is to define an interface for the target's functionality. Each function is implemented so that the function's command code and parameters are transmitted to the target with the help of the I2Cwrapper library.
 - `template_I2C_firmware.h` - Target firmware templates. Here, the most important part is injecting code into the command interpreter that will "unwrap" the controller function's command codes, react adequately, and, optionally, prepare a reply.
 
-Refer to the documentation within the templates' source code and to the existing modules for more details and illustration.
+Refer to the documentation within the templates' source code and to the [existing modules](src) for more details and illustration.
 
-### A note on messages and "(sub)units"
+### A Note on Messages and "(Sub)Units"
 
 All transmissions to the target device have a three byte header followed by an arbitrary number of zero or more parameter bytes.
 
 - [0] **CRC8 checksum**
 - [1] **command code**: Modules and the I2Cwrapper core use their own unique command code ranges, so that the command code will decide which module (if more than one is enabled) or if the core will interpret the command.
-- [3] **unit addressed**: If a target module enables I2C access to more than one instance of some hardware, e.g. multiple stepper or servo motors, the unit can be used to differentiate them. It is up to each module to decide if and how the unit is interpreted. Modules which don't need them because there is only one instance of their respective hardware (like e.g. the PinI2C module), can just ignore the unit and will have to live with the one byte wasted bandwidth per transmission.
+- [3] **unit addressed**: If a target module enables I2C access to more than one instance of some hardware, e.g. multiple stepper or servo motors, the unit can be used to differentiate them. It is up to each module to decide if and how the unit is interpreted. Modules which don't need them because there is only one instance of their respective hardware (like e.g. the `PinI2C` module), can just ignore the unit and will have to live with the one byte wasted bandwidth per transmission.
 
-# Supported platforms
+# Supported Platforms
```

Comment:
Looking forward to hear about your experiences. Possible caveats: The reset procedure is machine dependent, it needs to be rewritten to work without an ugly hardware reset. Also, in the sections with platform-specific preprocessor directives I did not yet systematically provide a catch-all #elsif clause. Concerning endians and sizof(int) etc. I think that I typeguarded most of the parameter transmissions, but it might be I missed one or two in the most recent modules.